src/interpreter/kernel/exec.js

```diff
@@ -703,6 +705,21 @@ export function executeStackFrame(
         return pop1();
       }
 
+      /**
+       * Memory operations
+       */
+      case "store": {
+        const memAddr = frame.originatingModule.memaddrs[0];
+        const memory = (frame.allocator.get(memAddr): Memory);
```

Comment:
In Flow this is actually not a cast, you want:

```js
const memory: Memory = frame.allocator.get(memAddr);
```

Reply:
thanks - need to read up a bit more on Flow :-)